sippy-ng/src/App.js

```diff
@@ -192,6 +195,27 @@ export default function App(props) {
     [setCookie]
   )
 
+  const testTableDBSourcePreference = cookies['testTableDBSource']
+  const [testTableDBSource, setTestTableDBSource] = React.useState(
+    testTableDBSourcePreference
+  )
+  const testTableDBSourceToggle = React.useMemo(
```

Comment:
Might be good to demo this to the team.  I wasn't expecting a live option to switch the new queries on.  I don't recall if we still have sippy_qe where I would expect this would fail currently and I'm curious if others thought it might be confusing or not to end users who discovered the settting and wonder what portions of the app it affects, etc.